### ISSUE TITLE
Support Apcsilmic Dot 1 (Snapdragon Qualcomm 7c/SC7180)

### ISSUE THREAD

**Issue by stevefan1999-personal** [aarch64-laptops/build#117]
The current status quo: Grub actually boots, but it displays garbled text after starting countdown -- a sign of video buffer corruption. 

After that, Linux unpacked itself in memory and then UEFI transitioned from Boot Service to Runtime Service -- after that point things went south -- No video output, no USB power, whole thing just turns off with no capability for debugging, probably an indication of both GOP video mode selection failure and a kernel panic at the same time, 

For FreeBSD it has the same fate, but on OpenBSD it surprising boots. However, I wasn't able to get through the installation menu since the USB keyboard lost connectivity, but at least the kernel did not go FUBAR.

Maybe I should start with ACPI entry dumps in Windows and see what we can do about it. I really want this thing to run Linux for a long time since it is a really capable machine for running as an IoT device center control terminal with its 4G and other network connectivity.

**Comment by stevefan1999-personal** [comment]
Here's in Armbian in action, notice this version of Grub did not have the corruption, but others (even latest Arch boot) do.

https://github.com/user-attachments/assets/db20d8d5-3aa7-4d20-bdc1-ef433128d584

**Comment by stevefan1999-personal** [comment]
Here's in OpenBSD. You can see that it boots well until USB was all gone. I would say we need to fix up the USB port enumerations in ACPI first. Should be reported back to OpenBSD people as well.

https://github.com/user-attachments/assets/69bdb101-5e84-4c3d-9d6d-cd40e1cddb8e

**Comment by stevefan1999-personal** [comment]
![image](https://github.com/user-attachments/assets/2b1d8840-4e96-41ab-b30a-44841dc21334)

Here's the list of images I tried so far, none of them works. OpenBSD is the one that went to most far

**Comment by acovaci** [comment]
Hi,

I'm also highly interested in this, however I have virtually 0 knowledge on how to even approach investigating this. 

But, if there's anything I can do to help, let me know.

**Comment by stevefan1999-personal** [comment]
> Hi,
> 
> I'm also highly interested in this, however I have virtually 0 knowledge on how to even approach investigating this. 
> 
> But, if there's anything I can do to help, let me know. 

Right, can you try to install more ARM64 UEFI iso images on that little box? I'm also looking for a way to dump the serial and boot log
